lib.php

```diff
@@ -8680,3 +8685,36 @@ function mention_id_array($postmessage="") {
     }
     return $id_array;
 }
+
+/**
+ * Function to delete all expired drafts based on the number of days set in the config
+ * @return void
+ */
+function delete_expired_custom_drafts() {
+    global $DB;
+
+    $daystopersistdrafts = false;
+
+    try {
+        $daystopersistdrafts = get_config('hsuforum', 'daystopersistdrafts');
```

Comment:
Motivation for this being in a try/catch. I do not think get_config throws an exception.

Reply:
It can throw a DML exception. Ordinarily it returns false, but just having a bit of extra error handling just to be sure in case it handles things differently being called from CLI vs web (not sure if it bootstraps it differently or something like that)